community/kernel/src/main/java/org/neo4j/unsafe/batchinsert/BatchInserterImpl.java

```diff
@@ -296,14 +299,15 @@ public Label apply( long from )
         schemaCache = new SchemaCache( new StandardConstraintSemantics(), schemaStore );
 
         Dependencies deps = new Dependencies();
-        deps.satisfyDependencies( fileSystem, config, logService, new NeoStoresSupplier()
-                        {
-                            @Override
-                            public NeoStores get()
-                            {
-                                return neoStores;
-                            }
-                        } );
+        deps.satisfyDependencies( fileSystem, config, logService, new NullJobScheduler(), new Monitors(),
```

Comment:
Should we instead prevent that kernel extension from loading? There's no means of doing that currently.